### ISSUE TITLE
GTA Trophies does not work together with CDR

### ISSUE THREAD

**Issue by TheSubPlayer** [Freakler/psp-gta_trophies#2]
When GTA Trophies and Cheatdevice Remastered are activated at the same time, CDR does not work (PSP 3000)

**Comment by Micsuit** [comment]
Yeah psp_trophies has some problems on real hardware. Seems to work fine on PPSSPP.
It probably has to do with some internal functions addresses changing because of CDR.

**Comment by daniemun** [comment]
This plugin has some problems in real hardware. In PPSSPP it seems to work perfectly.
My guess is that gta_trophies changes some adresses offsets which CDR doesn't like.
PSP's RAM ≠ PPSSPP's RAM.

**Comment by Freakler** [comment]
It's tested and working on all my PSPs / Vitas & even with GTA native and second analog plugin. What CFW and other plugins etc are you using?

**Comment by TheSubPlayer** [comment]
I use ARK-4 CFW
